comm/src/main/scala/coop/rchain/comm/discovery/GrpcKademliaRPCServer.scala

```diff
@@ -13,6 +13,9 @@ class GrpcKademliaRPCServer[F[_]: Monixable: Sync](
     lookupHandler: (PeerNode, Array[Byte]) => F[Seq[PeerNode]]
 ) extends KademliaGrpcMonix.KademliaRPCService {
 
+  // TODO: legacy code generates KademliaGrpcMonix methods with Task
+  //  so these methods cannot be abstracted over effect type
```

Comment:
Do you have an understanding how to go from protobuf definitions to code abstracted over effect type?